app/react/Library/components/TableCell.tsx

```diff
@@ -4,6 +4,7 @@ import { I18NLink } from 'app/I18N';
 import GeolocationViewer from 'app/Metadata/components/GeolocationViewer';
 import { LinkSchema, MetadataObjectSchema, PropertySchema } from 'shared/types/commonTypes';
 import MarkdownViewer from 'app/Markdown';
+// import moment from 'moment';
```

Comment:
Please remove commented imports, if we are sure that they are not used.

Reply:
Removed